docs-chef-io/content/inspec/resources/aws_api_gateway_vpc_link.md

```diff
@@ -0,0 +1,112 @@
++++
+title = "aws_api_gateway_vpc_link resource"
+platform = "aws"
+draft = false
+gh_repo = "inspec-aws"
+
+[menu.inspec]
+title = "aws_api_gateway_vpc_link"
+identifier = "inspec/resources/aws/aws_api_gateway_vpc_link resource"
+parent = "inspec/resources/aws"
++++
+
+Use the `aws_api_gateway_vpc_link` InSpec audit resource to test the properties of a single specific AWS APIGateway VPCLink.
+
+For additional information, including details on parameters and properties, see the [AWS documentation on AWS APIGateway VPCLink](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-vpclink.html).
+
+## Installation
+
+{{% inspec_aws_install %}}
+
+## Syntax
+
+Ensure that an VPC link exists.
+
+```ruby
+describe aws_api_gateway_vpc_link(vpc_link_id: 'VPC_LINK_ID') do
+  it { should exist }
+end
+```
+
+## Parameters
+
+`vpc_link_id` _(required)_
+
+: The identifier of the VpcLink.
+
+## Properties
+
```

Comment:
@soumyo13 - Do we need to include the field types of all these properties?

Reply:
No.